src/containers/User/UserRegister.tsx

```diff
@@ -1,166 +1,18 @@
-      cache.modify({
-        fields: {
-          users(existingUserRefs: UsersConnection, { readField }) {
-            const { user } = createdUser.createUser as CreateUserPayload
-            if (!user) return existingUserRefs
-
-            const newUserRef = cache.writeFragment({
-              data: user,
-              fragment: addNewUser,
-            })
-
-            // Quick safety check - if the new user is already
-            // present in the cache, we don't need to add it again.
-            if (
-              existingUserRefs.nodes.some((ref) => (ref ? readField('id', ref) === user.id : false))
-            ) {
-              return existingUserRefs
-            }
-
-            return [...existingUserRefs.nodes, newUserRef]
-          },
-        },
-      })
-    },
-    onCompleted: () => {
-      changeSnackback(submitedObject)
-      setTimeout(() => {
-        removeSnackbar()
-      }, 2000)
-    },
-  })
-
-  const updateUser = async () => {
-    try {
-      const { username, password, email } = newUser
-      await createUserMutation({
-        variables: {
-          username: username,
-          password: password,
-          email: email,
-        },
-      })
-    } catch (error) {
-      console.error(error)
-    }
-  }
-
-  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
-    event.preventDefault()
-    if (newUser.username != '' && newUser.password != '' && newUser.email != '') {
-      updateUser()
-    } else {
-      alert('Invalid user details')
-    }
-  }
-
-  return (
-    <Segment.Group>
-      <Segment>
-        <Form>
-          <Form.Field
-            id="form-input-username"
-            control={Input}
-            label="Username"
-            name="username"
-            content={newUser.username}
-            onChange={handleInputChange}
-          />
-          <Form.Field
-            id="form-input-password"
-            control={Input}
-            label="Password"
-            name="password"
-            content={newUser.password}
-            onChange={handleInputChange}
-          />
-          <Form.Field
-            id="form-input-email"
-            control={Input}
-            label="Email"
-            name="email"
-            content={newUser.email}
-            onChange={handleInputChange}
-          />
-          <Form.Field
-            id="form-input-terms-and-conditions"
-            control={Checkbox}
-            label="I agree to the Terms and Conditions"
-          />
-          <Form.Input control={Button} type="submit" content="Submit" onClick={handleSubmit} />
-        </Form>
-      </Segment>
-      <Segment>
-        <Message
-          success
-          hidden={!snackbar.showMessage}
-          header={snackbar.messageTitle}
-          content={snackbar.messageText}
-          error={snackbar.isErrorMessage}
-        />
-      </Segment>
-    </Segment.Group>
-  )
+  return <p>Re-directing to user registration application...</p>
```

Comment:
I tried going to the login page and stopping the server. It doesn't show any problems on the login page - which I think is fine, but the problem is when I click the `Create new account` icon, it is showing this message.

Maybe we just need to deal with this checking - server is unreachable in any page should display some icon or take the user to unreachable page. I think this can also be another issue. Do you have more thought around this?

Reply:
Good call -- I've added some simple error handling and now the page will display the network error message if there is one.

We can add some fancier error pages later on, I guess (along with 404 pages etc.)